lib/grape/validations/validators/coerce.rb

```diff
@@ -76,7 +76,7 @@ def coerce_value(val)
         converter.call(val)
 
       # Some custom types might fail, so it should be treated as an invalid value
-      rescue
+      rescue StandardError
```

Comment:
This is actually a behavior change. It's the right one though. Let's add it to UPGRADING?

Reply:
Separate PR ?

Reply:
Probably best. Remove it from this one?